### DISCUSSION TITLE
Set error manual for useFieldArray

### DISCUSSION THREAD

**Discussion by dinhmai74** [react-hook-form/react-hook-form#4035] [Q&A]
How can I setError with useFieldArray. I follow the docs but it's only work with normal field.

**Comment by bluebill1049** [comment]
can you share a codesandbox? `setError(FIeldName, error)`

make sure it's the field name not the key name of your field array.

**Reply by dinhmai74** [reply]
Thank you for the quick reply @bluebill1049 . It works fine in codesandbox. Don't know why it didn't work in my repo. I will find it myself later
